src/sentry/static/sentry/app/main.tsx

```diff
@@ -14,7 +17,10 @@ class Main extends React.Component {
   render() {
     return (
       <CacheProvider value={cache}>
-        <Router history={browserHistory}>{routes()}</Router>
+        <ThemeProvider theme={theme}>
+          <GlobalStyles theme={theme} />
```

Comment:
Moving these up to encompass unauthed parts of the app since they depend on emotion.